1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,59 @@
-function setAlarm() {}
+const colors = ['yellow', 'orange', 'pink', 'purple', 'aqua'];
+const changePageColor = (color) => document.querySelector('body').style.backgroundColor = color;
+const getRandomIndex = () => Math.floor(Math.random() * colors.length);
+
+
+function setAlarm() {
+  let inputElem = document.getElementById('alarmSet');
+  let timeRenderElem = document.getElementById('timeRemaining');
+  let numberSubmitted = inputElem.value;
+
```

Comment:
All good to me and seems to work correctly! 
It is assuming that the input value is always a number. If a non-numeric value is entered, the subsequent code will fail I believe. Can you think of handling this situation by adding a check?